vmdb/spec/requests/api/conditions_spec.rb

```diff
@@ -0,0 +1,132 @@
+#
+# REST API Request Tests - Conditions
+#
+# Condition primary collection:
+#   /api/conditions
+#
+# Condition subcollection:
+#   /api/policies/:id/conditions
+#
+require 'spec_helper'
+
+describe ApiController do
+
+  include Rack::Test::Methods
+
+  before(:each) do
+    init_api_spec_env
+  end
+
+  def app
+    Vmdb::Application
+  end
+
+  def create_conditions(count)
+    1.upto(count) do |i|
+      FactoryGirl.create(:condition, :name => "condition_#{i}", :description => "Condition #{i}")
```

Comment:
Also, fyi, instead of `1.upto(count) do |i|`, you can also do `count.times do |i|`